shared/actions/login/index.js

```diff
@@ -57,6 +73,7 @@ export function navBasedOnLoginState () :AsyncAction {
 function getAccounts (): AsyncAction {
   return dispatch => {
     const params: login_getConfiguredAccounts_rpc = {
+      ...makeWaitingHandler(dispatch),
```

Comment:
Isn't this just replicating what bindActionCreators does?

I think if you do
`waitingHandler: bindActionCreators(waitingForResponse, dispatch)`
it's the same thing and less of our own code